docs/content/en/docs/configuration/command-line.md

```diff
@@ -376,6 +448,27 @@ See also:
 
 ---
 
+## --update-status
+
+Indicates whether the ingress controller should update the Ingress status IP/hostname. Defaults to `true`.
```

Comment:
I copied this description from the CLI, but i'm unclear what it means exactly.  As a result this description could probably be improved.

Reply:
Updated based on new CLI help texts.